docs/ide-usage.md

```diff
@@ -43,7 +43,7 @@ The following plugins are needed:
    * Set the `root directory` to the location where the repo is cloned.
    * Click `Finish`.
 
-1. Configure the following project-specific settings (all can be found in `Project → Properties → ...`):
+1. Configure the following project-specific settings (all can be found in `Project → Properties → ...`, except for the HTML, CSS and XML settings which can be found at `Eclipse -> Preferences -> ...`):
```

Comment:
Sorry for my mistake, I used the wrong symbols in my comments, do replace `->` with `→`.